src/MizarWidgets/include/MizarWidgets/SamplingWithFrameTrackOutputWidget.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef MIZAR_WIDGETS_SAMPLING_WITH_FRAME_TRACK_OUTPUT_WIDGET_H_
+#define MIZAR_WIDGETS_SAMPLING_WITH_FRAME_TRACK_OUTPUT_WIDGET_H_
+
+#include <QWidget>
+
+#include "MizarData/SamplingWithFrameTrackComparisonReport.h"
+
+namespace Ui {
+class SamplingWithFrameTrackOutputWidget;
+}
+
+namespace orbit_mizar_widgets {
+
+class SamplingWithFrameTrackOutputWidget : public QWidget {
+  using Report = ::orbit_mizar_data::SamplingWithFrameTrackComparisonReport;
```

Comment:
Help me with C++ scoping again. So this alias is only within this class, right?

Reply:
Yes, and it is private.